src/templates/layouts/staffProfile/index.test.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { MediaImage } from '@/types/formatted/media'
+import { StaffProfile } from './index'
+
+describe('StaffProfile Component', () => {
+  const mediaImage: MediaImage = {
+    id: '3',
+    alt: 'Heather Steele outreach and community engagement specialist',
+    title: 'Heather Steele',
+    width: 151,
+    height: 227,
+    links: {
+      '1_1_square_medium_thumbnail': {
+        href: 'https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/styles/1_1_square_medium_thumbnail/public/2021-04/Zachary_Sage.jpg',
+        meta: {
+          linkParams: {},
+        },
+      },
+      '2_3_medium_thumbnail': {
+        href: 'https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/styles/2_3_medium_thumbnail/public/2021-04/Zachary_Sage.jpg',
+        meta: {
+          linkParams: {},
+        },
+      },
+    },
+  }
+  const completeBiography = {
+    value: 'public://2023-07/RCS Bio  Headshot.pdf',
+    url: 'https://dsva-vagov-staging-cms-files.s3.us-gov-west-1.amazonaws.com/2023-07/RCS%20Bio%20%20Headshot.pdf',
+  }
+  const mockProfile = {
+    firstName: 'Prachi',
+    lastName: 'Asher',
+    suffix: ', FACHE',
+    emailAddress: null,
+    phoneNumber: {
+      type: 'paragraph--phone_number',
+      id: '1234',
+      extension: null,
+      label: null,
+      phoneType: null,
+      number: '412-822-3537',
+    },
+    introText:
+      'Prachi V. Asher, FACHE, is the deputy director of VA Pittsburgh Healthcare System (VAPHS), effective May 9, 2022.',
+    description: 'Deputy Director',
+    body: '<p>Some biography text here.</p>',
+    completeBiographyCreate: true,
+    completeBiography: completeBiography,
+    photoAllowHiresDownload: true,
+    vamcOfficalName: 'VA Pittsburgh Healthcare System',
+    media: mediaImage,
+    menu: {
+      depth: 5,
+      link: { label: 'Prachi Asher', url: { path: '/' }, links: [] },
+      parent: {
+        label: 'Leadership',
+        links: [],
+        url: { path: '/leadership' },
+      },
+    },
+    id: '1',
+    type: 'staff',
+    published: true,
+    title: 'p',
+    lastUpdated: '2/2/25',
+  }
+
+  test('renders StaffProfile with correct data', () => {
+    render(<StaffProfile {...mockProfile} />)
+
+    expect(screen.getByText('Prachi Asher')).toBeInTheDocument()
+    expect(screen.getByText('Deputy Director')).toBeInTheDocument()
+    expect(
+      screen.getByText('VA Pittsburgh Healthcare System')
+    ).toBeInTheDocument()
+    expect(screen.getByText('Phone:')).toBeInTheDocument()
+    expect(screen.getByTestId('phone')).toBeInTheDocument()
+    expect(screen.getByText(mockProfile.introText)).toBeInTheDocument()
+    expect(screen.queryByText(/Download full size photo/)).toBeInTheDocument()
+    expect(screen.queryByText(/Download full bio/)).toBeInTheDocument()
```

Comment:
It's a nit, but we should stick with either `getBy*` in these tests.

Technically, we could just `screen.getByText('whatever')` without the `expect` and call it a day because they'll throw an error and fail the test if the element isn't found, but our eslint rules get mad because they're a little naive. (Which is fine. 🤷)

`queryBy*` can return null without throwing an error, so we need to use that for the `expect().not.toBeInTheDocument()`s.

Reply:
yep on the same page there, that was me rushing though tests